docs/Makefile

```diff
@@ -1,10 +1,10 @@
 # Minimal makefile for Sphinx documentation
 #
 
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = ewatercycle
```

Comment:
Is sphinxproj not needed anymore? Can't find docs on it in a quick search

Reply:
I did

```sh
# ewatercycle env active
# in temp dir
sphinx-quickstart
```

From which I copied the Makefile and make.bat, so indeed the SPHINXPROJ is no longer needed. I think it parses `project` from conf.py